src/zope/app/testing/dochttp.py

```diff
@@ -66,11 +66,10 @@ def dochttp(args=sys.argv[1:], default=None, output_fp=None):
         default = default_options
 
     options, args = parser.parse_args(default + args)
-    try:
-        directory, = args
-    except: # pragma: no cover
-        parser.print_help()
-        raise
+    if not args or not len(args) == 1:
```

Comment:
I don't think `optparse` can ever return `None` for `args`, so the `len()` check should suffice.